dist/dfu.js

```diff
@@ -134,6 +134,9 @@
             })
             .then(function() {
                 log("modeData written");
+                setTimeout(function() {  // TODO: Remove this when gattserverdisconnected event is implemented and possibly put a timeout in that event handler before resolving.
```

Comment:
We now have a situation in the node example where this function will resolve twice. I'm thinking we need this timeout and the disconnect eventhandler above to call the same function which checks whether the server is connected and disconnects+resolves if it is.